### DISCUSSION TITLE
Quit if last main editor window is closed

### DISCUSSION THREAD

**Discussion by mehalter** [folke/edgy.nvim#12] [Ideas]
With this being a canonical way to organize "sidebars" and basically temporary/ephemeral/pinned windows, a lot of the times it can be safe to assume if the last main editor window is closed (i.e. only edgy managed windows remain) then the user actually wants the editor to quit. I think this would be a nice feature to add or even just a snippet in the README using the exposed edgy API to create an `autocmd` the user could use to achieve this.

**Comment by folke** [comment]
Good point. Think I';; make this configurable.
Right now I actually recreate the main window if it's closed when a sidebar is still open.

**Comment by folke** [comment]
Added `config.exit_when_last` (disabled by default)

**Reply by mehalter** [reply]
Thanks! I did some basic testing and it works sometimes, but not all the time. Don't have time today to make a detailed bug report, but I figured I would just give you a heads up. I will try to find the time to look into what might be causing it to open an actual bug report.
